### PR TITLE
Remove the OpenGL backend on iOS

### PR DESCRIPTION
**Pull request by zanderso**

### CODE REVIEW

File: shell/platform/darwin/ios/BUILD.gn [hunk 1]
```diff
@@ -185,7 +175,6 @@ source_set("flutter_framework_source") {
     "AudioToolbox.framework",
```
Comment (chinmaygarde) [inline shell/platform/darwin/ios/BUILD.gn:175]: Completely unrelated, but why in the world do we link in AudioToolbox? I think this is code I wrote my first week at Google :/

Comment (zanderso) [reply shell/platform/darwin/ios/BUILD.gn:175]: Let's find out https://github.com/flutter/engine/pull/34916

**Review by chinmaygarde — APPROVED** [review]
Looks good to me and this covers all the files I can think to get rid of.

File: shell/platform/darwin/ios/rendering_api_selection.mm [hunk 2]
```diff
@@ -52,16 +52,15 @@ IOSRenderingAPI GetRenderingAPIForProcess(bool force_software) {
 #if TARGET_OS_SIMULATOR
```
Comment (jmagman) [inline shell/platform/darwin/ios/rendering_api_selection.mm:52]: The comment right above this (which GitHub isn't letting me comment on) still references OpenGL

File: shell/platform/darwin/ios/BUILD.gn (hunk 1 above)
Comment (jason-simmons) [reply shell/platform/darwin/ios/BUILD.gn:175]: The platform plugin interface does provide an API for playing a sound:
https://github.com/flutter/engine/blob/main/shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm#L96

File: shell/platform/darwin/ios/rendering_api_selection.mm (hunk 2 above)
Comment (zanderso) [reply shell/platform/darwin/ios/rendering_api_selection.mm:53]: Reworded the comment a bit.

**Review by jmagman — APPROVED** [review]
Screen Shot 2022-07-26 at 11 51 45 AM
🥳 

LGTM, let's see how this does on a full TAP run...

**Pull request merged**